README.rst

```diff
@@ -7,15 +7,15 @@ Django Include
 
 ORM extensions for performance-conscious perfectionists.
 
-Django-include provides `select_related` functionality for Many-to-X relation.
+Django-include provides ``select_related`` functionality for Many-to-X relations.
 
 
 Requirements
 ============
 
 Python 2.7 or 3.4+, Django 1.9+, and any SQL server with support for JSON aggregations.
 
-Currently tested against Postgres. May work with SQLite with the JSON1 extension.
+Currently tested against Postgres 9.6. May work with SQLite with the JSON1 extension.
```

Comment:
That's probably true, but I included this because 9.6 is the only one we're officially testing against in CI. Everything else is "use at your own risk".